platform/mellanox/mlnx-ffb.sh

```diff
@@ -49,9 +49,9 @@ check_sdk_upgrade()
             break;
         }
 
-        ISSU_CHECK_CMD="docker exec -t syncd issu --check ${NEXT_SDK_VERSION}"
-
-        ${ISS_CHECK_CMD} > /dev/null && CHECK_RESULT="${FFB_SUCCESS}"
+        # TODO: Place a call to SDK check script
+        # for now assume check succeeded
```

Comment:
Did you add a tab here? Please set you editor to expand white spaces and tab stop at 4 to avoid future issues.

Reply:
on line 53 there are only spaces. This came from another commit that is part of #2384, in this PR I removed these lines